<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>tool_check_time</class>
 <widget class="QMainWindow" name="tool_check_time">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>395</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>600</width>
    <height>395</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>600</width>
    <height>395</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>校时</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QMainWindow
{
	
	background-color: rgb(255, 255, 255);
}</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>40</y>
      <width>601</width>
      <height>291</height>
     </rect>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>20</y>
       <width>93</width>
       <height>201</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QGroupBox
{
	border : none;
}</string>
     </property>
     <property name="title">
      <string/>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>NTP设置</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>NTP服务器</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>NTP端口</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>更新周期</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QGroupBox" name="groupBox_3">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>20</y>
       <width>363</width>
       <height>251</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>9</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">border: none;</string>
     </property>
     <property name="title">
      <string/>
     </property>
     <layout class="QGridLayout" name="gridLayout_4">
      <item row="0" column="0" colspan="2">
       <widget class="QGroupBox" name="groupBox_4">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>32</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>130</width>
          <height>32</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">QRadioButton::indicator:checked
{
	border-image:url(:/png/shuncom_ico/png_ico/单选框.png)
}
</string>
        </property>
        <property name="title">
         <string/>
        </property>
        <layout class="QGridLayout" name="gridLayout_2">
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <property name="spacing">
          <number>0</number>
         </property>
         <item row="0" column="0">
          <widget class="QRadioButton" name="radioButton">
           <property name="text">
            <string>是</string>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>51</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item row="0" column="2">
          <widget class="QRadioButton" name="radioButton_2">
           <property name="text">
            <string>否</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item row="1" column="0" colspan="3">
       <widget class="QLineEdit" name="lineEdit">
        <property name="minimumSize">
         <size>
          <width>260</width>
          <height>32</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>260</width>
          <height>32</height>
         </size>
        </property>
       </widget>
      </item>
      <item row="2" column="0" colspan="3">
       <widget class="QLineEdit" name="lineEdit_2">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="minimumSize">
         <size>
          <width>260</width>
          <height>32</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>260</width>
          <height>32</height>
         </size>
        </property>
        <property name="text">
         <string>123</string>
        </property>
       </widget>
      </item>
      <item row="2" column="3">
       <widget class="QLabel" name="label_6">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>32</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>32</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">color: rgb(200, 201, 204);</string>
        </property>
        <property name="text">
         <string>(0~65535)</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="3" column="0" colspan="3">
       <widget class="QGroupBox" name="groupBox_5">
        <property name="minimumSize">
         <size>
          <width>260</width>
          <height>32</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>260</width>
          <height>32</height>
         </size>
        </property>
        <property name="title">
         <string/>
        </property>
        <layout class="QGridLayout" name="gridLayout_3">
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <property name="horizontalSpacing">
          <number>0</number>
         </property>
         <item row="0" column="0">
          <widget class="QLineEdit" name="lineEdit_3">
           <property name="minimumSize">
            <size>
             <width>205</width>
             <height>30</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>205</width>
             <height>30</height>
            </size>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QLabel" name="label_5">
           <property name="minimumSize">
            <size>
             <width>52</width>
             <height>30</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>52</width>
             <height>30</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">background-color: rgb(247, 248, 250);
color: rgb(100, 101, 102);</string>
           </property>
           <property name="text">
            <string>分钟</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item row="3" column="3">
       <widget class="QLabel" name="label_7">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>32</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>32</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">color: rgb(200, 201, 204);</string>
        </property>
        <property name="text">
         <string>(0~65535)</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QPushButton" name="pushButton_2">
        <property name="minimumSize">
         <size>
          <width>76</width>
          <height>32</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>76</width>
          <height>32</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">/*按钮普通态*/
QPushButton
{

    /*字体颜色*/ 
	color: rgb(50, 50, 51);
 	/*背景颜色*/  
    background-color: rgb(255, 255, 255);
    /*边框圆角半径为8像素*/ 
    border-radius:2px;
	width:65px;
}
 
/*按钮停留态*/
QPushButton:hover
{
   /*背景颜色*/  
    background-color: rgb(85, 170, 255);
}
 
/*按钮按下态*/
QPushButton:pressed
{
    /*背景颜色*/  
 	background-color:rgb(170 , 255 , 255);
    /*左内边距为3像素，让按下时字向右移动3像素*/  
    padding-left:3px;
    /*上内边距为3像素，让按下时字向下移动3像素*/  
    padding-top:3px;
}</string>
        </property>
        <property name="text">
         <string>取消</string>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="QPushButton" name="pushButton">
        <property name="minimumSize">
         <size>
          <width>76</width>
          <height>32</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>76</width>
          <height>32</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>Microsoft Yahei</family>
          <pointsize>9</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">/*按钮普通态*/
QPushButton
{


    /*字体颜色*/ 
	color: rgb(255, 255, 255);
 	/*背景颜色*/  
	background-color: rgb(21, 91, 212);
    /*边框圆角半径为8像素*/ 
    border-radius:0px;
	width:65px;
}
 
/*按钮停留态*/
QPushButton:hover
{
   /*背景颜色*/  
    background-color: rgb(85, 170, 255);
}
 
/*按钮按下态*/
QPushButton:pressed
{
    /*背景颜色*/  
 	background-color:rgb(170 , 255 , 255);
    /*左内边距为3像素，让按下时字向右移动3像素*/  
    padding-left:3px;
    /*上内边距为3像素，让按下时字向下移动3像素*/  
    padding-top:3px;
}</string>
        </property>
        <property name="text">
         <string>确定</string>
        </property>
       </widget>
      </item>
      <item row="4" column="2" colspan="2">
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>170</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>340</y>
      <width>157</width>
      <height>19</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="label_8">
       <property name="styleSheet">
        <string notr="true">color: rgb(98, 98, 102);</string>
       </property>
       <property name="text">
        <string>已选择设备数:</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_9">
       <property name="minimumSize">
        <size>
         <width>50</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>50</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources>
  <include location="../../shuncom_qrc.qrc"/>
 </resources>
 <connections/>
</ui>
